code/modules/food_and_drinks/food/foods/sandwiches.dm

```diff
@@ -130,15 +143,187 @@
 	name = "SuperBite burger"
 	desc = "This is a mountain of a burger. FOOD!"
+	icon = 'icons/obj/food/burgerbread.dmi'
 	icon_state = "superbiteburger"
 	filling_color = "#CCA26A"
 	bitesize = 7
 	list_reagents = list("nutriment" = 40, "vitamin" = 5)
 	tastes = list("bun" = 4, "meat" = 2, "cheese" = 2, "type two diabetes" = 10)
 
+/obj/item/reagent_containers/food/snacks/crazyburger
+	name = "crazy hamburger"
+	desc = "This looks like the sort of food that a demented clown in a trenchcoat would make."
+	icon = 'icons/obj/food/burgerbread.dmi'
+	icon_state = "crazyburger"
+	filling_color = "#E3D681"
+	bitesize = 3
+	list_reagents = list("nutriment" = 10, "vitamin" = 2, "capsaicin" = 3, "condensedcapsaicin" = 2)
+	tastes = list("bun" = 2, "meat" = 4, "cheese" = 2, "beef soaked in chili" = 3, "a smoking flare" = 2)
+
+/obj/item/reagent_containers/food/snacks/ppattyred
+	name = "red pretty patty"
+	desc = "Perfect for hiding the fact that it's burnt to a crisp."
+	icon = 'icons/obj/food/burgerbread.dmi'
+	icon_state = "ppatty-red"
+	filling_color = "#D63C3C"
+	bitesize = 2
+	list_reagents = list("nutriment" = 7, "protein" = 1)
+	tastes = list("bun" = 2, "meat" = 2, "red" = 2)
+
+/obj/item/reagent_containers/food/snacks/ppattyorange
+	name = "orange pretty patty"
+	desc = "Contains 0% juice."
+	icon = 'icons/obj/food/burgerbread.dmi'
+	icon_state = "ppatty-orange"
+	filling_color = "#FFA500"
+	bitesize = 2
+	list_reagents = list("nutriment" = 7, "protein" = 1)
+	tastes = list("bun" = 2, "meat" = 2, "orange" = 2)
+
+/obj/item/reagent_containers/food/snacks/ppattyyellow
+	name = "yellow pretty patty"
+	desc = "Bright to the last bite."
+	icon = 'icons/obj/food/burgerbread.dmi'
+	icon_state = "ppatty-yellow"
+	filling_color = "#FFFF00"
+	bitesize = 2
+	list_reagents = list("nutriment" = 7, "protein" = 1)
+	tastes = list("bun" = 2, "meat" = 2, "yellow" = 2)
+
+/obj/item/reagent_containers/food/snacks/ppattygreen
+	name = "green pretty patty"
+	desc = "It's not tainted meat, it's painted meat!"
+	icon = 'icons/obj/food/burgerbread.dmi'
+	icon_state = "ppatty-green"
+	filling_color = "#00FF00"
+	bitesize = 2
+	list_reagents = list("nutriment" = 7, "protein" = 1)
+	tastes = list("bun" = 2, "meat" = 2, "green" = 2)
+
+/obj/item/reagent_containers/food/snacks/ppattyblue
+	name = "blue pretty patty"
+	desc = "Is this blue rare?"
+	icon = 'icons/obj/food/burgerbread.dmi'
+	icon_state = "ppatty-blue"
+	filling_color = "#0000FF"
+	bitesize = 2
+	list_reagents = list("nutriment" = 7, "protein" = 1)
+	tastes = list("bun" = 2, "meat" = 2, "blue" = 2)
+
+/obj/item/reagent_containers/food/snacks/ppattypurple
+	name = "purple pretty patty"
+	desc = "Regal and low class at the same time."
+	icon = 'icons/obj/food/burgerbread.dmi'
+	icon_state = "ppatty-purple"
+	filling_color = "#800080"
+	bitesize = 2
+	list_reagents = list("nutriment" = 7, "protein" = 1)
+	tastes = list("bun" = 2, "meat" = 2, "purple" = 2)
+
+/obj/item/reagent_containers/food/snacks/ppattywhite
```

Comment:
Could you make the pretty patties use inheritence? Maybe having the white pretty patty be the base because of the default filling color
```suggestion
/obj/item/reagent_containers/food/snacks/ppatty/white
```